webpack.dev.config.ts

```diff
@@ -26,7 +26,7 @@ module.exports = merge(common, {
     filename: '[name].bundle.js',
     path: path.resolve(__dirname, 'webpack_bundles')
   },
-  devtool: 'inline-source-map',
+  devtool: 'eval',
```

Comment:
Should we go with `eval-cheap-source-map` here, to have at least line numbers?

Reply:
Umm, i think if we are optimizing the build according to time, we should do it completely. Also given that developers can use `--source_maps` flag if they want.

Reply:
Okay